person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permession
+
+  def initialize(age, name = 'unkown', parent_permession: true)
+    @id = rand
+    @name = name
+    @parent_permession = parent_permession
+    @age = age
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permession
```

Comment:
- [ ] for the can_use_services? kindly use the of_age? method for testing 🙏 

 def can_use_services? 
    of_age?() || @parent_permission
  end